### ISSUE TITLE
Use an icon and text as labels for arrow buttons

### ISSUE THREAD

**Issue by ciaoaaron** [RubberWheelz/Enabled-Gaming-Website#7]
https://github.com/RubberWheelz/Enabled-Gaming-Website/blob/main/index.html#L66-L67

The buttons here are not accessible. The screen reader would read it as "&gt;"

Inspect the UI on amazon and/or other popular sites and check out how they handle this situation.
